frontend/style.css

```diff
@@ -17,17 +17,22 @@ h4{
 }
 section{
    
-    width:100%;
-    height:800px;
-    display: flex;
-    justify-content: center;
+   display: flex;
+   justify-content: center;
     position: relative;
-    top:80px;
-    left:20px
+    top:50px;
+    left:-120px;
+   width:1000px;
+   
 }
 img{
-    width:300px;
-    
+  
+   position:absolute;
```

Comment:
Avoid using position absolute. It's not a good fix. Using position absolute will result in a page that is not responsive and will adjust poorly to different sized screens.